### ISSUE TITLE
Implement Hydra 70 rocket submodels on LAU-68.

### ISSUE THREAD

**Issue by JMaverick16** [NikolaiVChr/f16#102]
Once reaching 7 missiles left, when pulling trigger it still stated rockets away but didn't fire and also unable to fire gun. (This has happened only once).
Also sometimes earlier when pressing trigger although saying rockets away the Hydra never fired
Code is also acting like gun round hits and not like "exploded at N distance from ---", (probably can't make it other way)
I also have to understand how to properly make disappear the rocket from the LAU-68 after being fired like the A-10 does.
Also noticed that at times it fires 2 rockets instead of one
Can't explain why though, it's a bit random. Sometimes 2 most other 1.
Sound doesn't always play when a rocket is launched.
(Copy/paste from issues posted on Discord).

**Comment by NikolaiVChr** [comment]
To make rockets go away in 3D, just use the ai/submodels/submodel[4]/count property.
Use <greater-than> to check if a rocket should be shown.

**Comment by NikolaiVChr** [comment]
Did some commits, let me know which issues is still there.

**Comment by JMaverick16** [comment]
The "7 rockets unfireable" happens rarely and happened after respawning, probably related.
Gun works last time I checked when using rockets.
Still happens that at times when pressing trigger rockets don't fire (especially when done with minimal delay) or states "Rockets away" without displaying the rocket (I've also had times where rocket fires but explosion effect is missing).
Is it possible to add a range where it could hit (within 25m?) Else so far they have been pretty unreliable against targets (4/5 hits out of 28).
I've tried to make Hydras disappear from the LAU after firing w/ said property but without success (that's me not knowing how to do it properly).
Issue with multiple rockets per trigger seems disappeared but have to check once again to be sure.
Sounds now plays all the time.

**Comment by NikolaiVChr** [comment]
> Is it possible to add a range where it could hit

No idea what you mean.

> "7 rockets unfireable"

This still an issue? If it happens after respawn, probably nothing I can do about it, but if it happens before again, please tell.

>pressing trigger rockets don't fire

This might be a submodel issue, if thats the case nothing I can do to fix it. Submodels are kinda very unreliable, you odont notice for a gun, but for rocket you do.

> Hydras disappear from the LAU

For pylon 3 left check value of ai/submodels/submodel[4]/count.
For first rocket if greater than 6 then show.
For second rocket if greater than 5 then show.
etc etc..

**Comment by JMaverick16** [comment]
Like for other weapons, max hit range: <max-report-distance type="double">25</max-report-distance>. Dunno if it can be put in the submodel code directly or needs to be done differently.

So far I haven't had it for some time, will test again after respawn or reair to see it if is still affected.

Alright.

I've tried something like that if you check the LAU-68.xml code but wouldn't work, maybe I've made it the wrong way or should have implemented it differently (I'd do a single LAU-68.xml file for each pylon but not so smart.
